### ISSUE TITLE
Improve: we should solve build warmings

### ISSUE THREAD

**Issue by yiwenlong** [Hyperledger-TWGC/tape#169]
**Describe the bug**
I got the unexpected following information when build tape by execute `make`.

**Build Logs**
```
Building tape program......
go build -tags "" -ldflags "-X tape/pkg/infra.Version=0.0.2 -X tape/pkg/infra.CommitSHA=9d8343d -X 'tape/pkg/infra.BuiltTime=Sun Apr  4 08:00:42 CST 2021'" -gcflags=-m ./cmd/tape
# tape/cmd/tape
cmd/tape/main.go:60:24: inlining call to os.(*File).WriteString
cmd/tape/main.go:64:24: inlining call to os.(*File).WriteString
cmd/tape/main.go:69:13: inlining call to fmt.Printf
cmd/tape/main.go:72:14: inlining call to logrus.(*Logger).Infof
cmd/tape/main.go:73:14: inlining call to logrus.(*Logger).Infof
cmd/tape/main.go:32:19: inlining call to logrus.New
cmd/tape/main.go:33:17: inlining call to logrus.(*Logger).SetLevel
cmd/tape/main.go:36:19: inlining call to logrus.(*Logger).SetLevel
cmd/tape/main.go:40:30: inlining call to kingpin.MustParse
cmd/tape/main.go:40:30: inlining call to kingpin.Fatalf
cmd/tape/main.go:43:13: inlining call to fmt.Printf
cmd/tape/main.go:52:15: inlining call to logrus.(*Logger).Error
cmd/tape/main.go:58:16: rate does not escape
cmd/tape/main.go:58:31: leaking param: burst
cmd/tape/main.go:58:43: leaking param: logger
cmd/tape/main.go:60:24: ([]byte)(os.s) does not escape
cmd/tape/main.go:64:24: ([]byte)(os.s) does not escape
cmd/tape/main.go:69:79: int64(*rate) escapes to heap
cmd/tape/main.go:69:93: int64(*burst) escapes to heap
cmd/tape/main.go:69:13: []interface {} literal does not escape
cmd/tape/main.go:72:50: *rate escapes to heap
cmd/tape/main.go:72:14: []interface {} literal does not escape
cmd/tape/main.go:73:14: []interface {} literal does not escape
cmd/tape/main.go:32:19: &logrus.Logger literal escapes to heap
cmd/tape/main.go:32:19: new(logrus.TextFormatter) escapes to heap
cmd/tape/main.go:32:19: make(logrus.LevelHooks) escapes to heap
cmd/tape/main.go:40:30: []interface {} literal does not escape
cmd/tape/main.go:48:22: ... argument does not escape
cmd/tape/main.go:48:23: fullCmd escapes to heap
cmd/tape/main.go:52:15: []interface {} literal does not escape
<autogenerated>:1: .this does not escape
```

**Expected behavior**
The warming logs should be solve.

**Comment by guoger** [comment]
thanks for spotting this. It's not **warning**, but logs from compiler optimizations. we should remove `-m` from https://github.com/Hyperledger-TWGC/tape/blob/1201876afd0a555d07a2f9cc673ee8c10ddcf01e/Makefile#L40 to suppress these messages. Could you submit a PR?

**Comment by yiwenlong** [comment]
> thanks for spotting this. It's not **warning**, but logs from compiler optimizations. we should remove `-m` from
> 
> https://github.com/Hyperledger-TWGC/tape/blob/1201876afd0a555d07a2f9cc673ee8c10ddcf01e/Makefile#L40
> 
> to suppress these messages. Could you submit a PR?

OK

**Comment by guoger** [comment]
fixed by #171
